README.md

```diff
@@ -54,13 +58,37 @@ The following tables contains an overview (GC = Graph Collection, G = Logical Gr
 
 #### Unary logical graph operators (one graph as input):
 
-| Operator      | Output | Output description                                           | Impl |
-|:--------------|:-------|:-------------------------------------------------------------|:----:|
-| Aggregation   | G      | Graph with result of an aggregate function as a new property | Yes  |
-| Matching      | GC     | Graphs that match a given graph pattern                      | Yes  |
-| Transformation| G      | Graph with transformed (graph, vertex, edge) data            | Yes  |
-| Grouping      | G      | Structural condense of the input graph                       | Yes  |
-| Subgraph      | G      | Subgraph that fulfils given vertex and edge predicates       | Yes  |
+| Operator                    | Output | Output description                                           | Impl |
+|:----------------------------|:-------|:-------------------------------------------------------------|:----:|
+| Aggregation                 | G      | Graph with result of an aggregate function as a new property | Yes  |
+| Matching                    | GC     | Graphs that match a given graph pattern                      | Yes  |
```

Comment:
I would split Matching in "Cypher" and "Query" to retain the naming of the operators itself.
Cypher: Graphs that match a given Cypher graph pattern
Query: Graphs that match a given Cypher-based GDL graph pattern